frontend/packages/ux-editor/src/components/config/editModal/EditOptions/OptionTabs/EditTab/OptionListEditor/OptionListEditor.tsx

```diff
@@ -1,184 +1,69 @@
-}: EditLibraryOptionListEditorModalProps): React.ReactNode {
-  const { t } = useTranslation();
-  const { org, app } = useStudioEnvironmentParams();
-  const { doReloadPreview } = usePreviewContext();
-  const { mutate: updateOptionList } = useUpdateOptionListMutation(org, app);
-  const [localOptionList, setLocalOptionList] = useState<Option[]>(optionsList);
-  const editorTexts: CodeListEditorTexts = useOptionListEditorTexts();
-  const modalRef = createRef<HTMLDialogElement>();
-
-  const optionListHasChanged = (options: Option[]): boolean =>
-    JSON.stringify(options) !== JSON.stringify(localOptionList);
-
-  const handleBlurAny = (options: Option[]) => {
-    if (optionListHasChanged(options)) {
-      updateOptionList({ optionListId: optionsId, optionsList: options });
-      setLocalOptionList(options);
-      doReloadPreview();
-    }
-  };
-
-  const handleClose = () => {
-    modalRef.current?.close();
-  };
-
-  return (
-    <>
-      <StudioProperty.Button
-        value={label}
-        title={t('ux_editor.options.option_edit_text')}
-        property={t('ux_editor.modal_properties_code_list_button_title_library')}
-        onClick={() => modalRef.current.showModal()}
-      />
-      <StudioModal.Dialog
-        ref={modalRef}
-        className={classes.editOptionTabModal}
-        contentClassName={classes.content}
-        closeButtonTitle={t('general.close')}
-        heading={t('ux_editor.modal_add_options_code_list')}
-        onInteractOutside={handleClose}
-        onBeforeClose={handleClose}
-        footer={
-          <StudioAlert severity={'warning'} size='sm'>
-            {t('ux_editor.modal_properties_code_list_alert_title')}
-          </StudioAlert>
+      case 'success': {
+        if (optionsLists[component.optionsId] !== undefined) {
+          return (
+            <LibraryOptionsEditor
+              component={component}
+              optionsList={optionsLists[component.optionsId]}
+              handleDelete={handleDelete}
+            />
+          );
         }
-      >
-        <StudioCodeListEditor
-          codeList={localOptionList}
-          onBlurAny={handleBlurAny}
-          texts={editorTexts}
-        />
-      </StudioModal.Dialog>
-    </>
-  );
-}
-
-type EditManualOptionListEditorModalProps = {
-  label: string;
-} & Pick<IGenericEditComponent<SelectionComponentType>, 'component' | 'handleComponentChange'>;
-
-function EditManualOptionListEditorModal({
-  label,
-  component,
-  handleComponentChange,
-}: EditManualOptionListEditorModalProps): React.ReactNode {
-  const { t } = useTranslation();
-  const modalRef = useRef<HTMLDialogElement>(null);
-  const editorTexts = useOptionListEditorTexts();
-
-  const handleBlurAny = (options: Option[]) => {
-    if (component.optionsId) {
-      delete component.optionsId;
+        return (
+          <StudioAlert severity={'info'} size='sm'>
+            {t('ux_editor.options.tab_option_list_alert_title')}
+          </StudioAlert>
```

Comment:
Something seems a bit off with the placement of this. This should be shown in the bottom in the AddOptionList component if the options are neither manual or an existing optionsId. In my opinion 😄